keyboards/handwired/frame_48/keymaps/default/readme.md

```diff
@@ -0,0 +1 @@
+# The default keymap for the Frame 48
```

Comment:
This file should be removed. Same for `keyboards/handwired/frame_48/rev1/readme.md`